Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2826 -- Granular counts on the front end

Resolves [EREGCSC-2826](https://jiraent.cms.gov/browse/EREGCSC-2826)

**Description**

Adds result counts to the Document Type checkbox labels on Search and Subjects page.

**This pull request changes:**

- Creates `useCounts` composable to fetch counts from `content-search/counts`
- Adds `makeLabel` and `makeCount` helper methods to create Document Type checkbox label and counts
- Updates Cypress fixtures and tests
- Updates ESLint config and adds Prettier config for better linting and formatting

**Steps to manually verify this change:**

1. Green check marks
2. Visit experimental deployment's [Search](https://dbaihw1fw2.execute-api.us-east-1.amazonaws.com/dev1419/search) and [Subjects](https://dbaihw1fw2.execute-api.us-east-1.amazonaws.com/dev1419/subjects/) pages
3. Ensure that accurate counts appear beside both selected and unselected checkboxes in the Document Type selector for any kind of search using any or all filters
